packages/gatsby-theme-carbon/src/components/Switcher/Switcher.js

```diff
@@ -16,12 +17,10 @@ const Switcher = ({ children }) => {
       : setHeight(0);
   }, [listRef, switcherIsOpen]);
 
-  const lgBreakpoint = window.matchMedia('(min-width: 1056px)'); // returns true if screen width matches media query
+  const lgBreakpoint = useMedia('min-width: 1056px');
   const styles = {
-    container: (isLg, isOpen) => ({
-      /* eslint no-unused-expressions: [2, { allowShortCircuit: true, allowTernary: true }] */
-      maxHeight: !isLg && isOpen ? '100%' : `${height}px`,
-    }),
+    /* eslint no-unused-expressions: [2, { allowShortCircuit: true, allowTernary: true }] */
```

Comment:
What do you think about doing this in a way that gets around the eslint ignore? We could get rid of the `styles` object as well since it only has one property.

```js
let maxHeight = '100%';
if (!largeBreakpoint && switcherIsOpen) {
	maxHeight = `${height}px`;
}

//later
style={{maxHeight}}
```

Reply:
@vpicone so doing it that way throws the same eslint error.

Reply:
I tried this and still got the same error.

```js
  let maxHeight = `${height}px`;
  if (!lgBreakpoint && switcherIsOpen) {
    maxHeight = "100%";
  }
```

Reply:
Sorry it took me so long to get to this, lets take a look at it together whenever you have a sec.